db/migrate/20200511024721_backfill_position_for_books.rb

```diff
@@ -0,0 +1,26 @@
+class BackfillPositionForBooks < ActiveRecord::Migration[6.0]
+  BOOK_SLUGS_IN_ORDER = %w[no-wall-they-can-build
+                           from-democracy-to-freedom
+                           contradictionary
+                           work
+                           expect-resistance
+                           recipes-for-disaster
+                           days-of-war-nights-of-love
+                           off-the-map
+                           no-habra-muro-que-nos-pare
+                           da-democracia-a-liberdade
+                           trabalho-edicao-resumida-de-emergencia
+                           espere-resistencia
+                           receitas-para-o-desastre
+                           dias-de-guerra-noites-de-amor].freeze
+
+  def up
+    BOOK_SLUGS_IN_ORDER.each_with_index do |slug, index|
+      Book.find_by(slug: slug)&.update!(position: index)
+    end
+  end
+
+  def down
+    # no op
+  end
+end
```

Comment:
Instead of a data migration, would you mind just providing that script on the PR description?